src/main/java/org/elasticsearch/index/translog/Translog.java

```diff
@@ -145,36 +115,17 @@
     public Path[] locations();
 
     /**
-     * Returns the translog file with the given id as a Path. This
-     * will return a relative path.
-     */
-    Path getPath(long translogId);
-
-    /**
      * return stats
      */
     TranslogStats stats();
 
-    /**
-     * Returns the largest translog id present in all locations or <tt>-1</tt> if no translog is present.
+    /** notifies the translog that translogId was committed into lucene, allowing it
+     * to release all previous translogs
```

Comment:
To clarify, does this mean that if translog "6" is committed, that it also means all the data from translogs "5" and "4" exists? Or just "6"?